querybook/server/lib/export/base_exporter.py

```diff
@@ -160,6 +161,13 @@ def _get_statement_execution_download_url(self, statement_execution_id: int):
                     return reader.get_download_url()
         return None
 
+    def _get_query_execution_url_by_statement_id(self, statement_execution_id: int):
+        statement_execution = logic.get_statement_execution_by_id(
+            statement_execution_id
+        )
+        query_execution_id = statement_execution.query_execution_id
+        return f"{QuerybookSettings.PUBLIC_URL}/query_execution/{query_execution_id}"
```

Comment:
I think there is an `env` in the url.  e.g. querybook.pinadmin.com/data/query_execution

Reply:
thanks, good catch